accname/aria-owns.html

```diff
@@ -0,0 +1,236 @@
+<!doctype html>
+<html>
+
+<head>
+  <meta charset="utf-8">
+  <title>Accname tests for assorted combinations of aria-owns, aria-hidden, and host language hiding techniques</title>
+  <script src="/resources/testharness.js"></script>
+  <script src="/resources/testharnessreport.js"></script>
+  <script src="/resources/testdriver.js"></script>
+  <script src="/resources/testdriver-vendor.js"></script>
+  <script src="/resources/testdriver-actions.js"></script>
+  <script src="/wai-aria/scripts/aria-utils.js"></script>
+  <style>
+    [aria-hidden="true"] {
+      outline: 3px dashed darkred;
+    }
+
+    [id^=child] {
+      font-weight: bold;
+      background-color: yellow;
+    }
+  </style>
+</head>
+
+<body>
+
+  <h1>Accname tests for assorted combinations of <code>aria-owns</code>, <code>aria-hidden</code>, and host language hiding techniques</h1>
+  <p><span aria-hidden="true">Text enclosed in a dashed red outline</span> is <code>aria-hidden="true"</code>.</p>
+  <p><span id="child-simulated">Bold text with a yellow background</span> is the target of another element’s <code>aria-owns</code>.</p>
+
+  <h2>Child is moved, nothing is hidden</h2>
+  <p>Expected: <q>I love cats and dogs I like birds</q></p>
+  <button
+    class="ex-label"
+    data-testname="Child is moved, nothing is hidden"
```

Comment:
Updates to the test name might also lend clarity. It's difficult to understand the expectations from the CI results comparison page too.

Reply:
e.g. why does the "Child is moved, nothing is hidden" test expect "I love cats and dogs I like birds"?